sdk/storage/azure-storage-blob/azure/storage/blob/_lease.py

```diff
@@ -6,15 +6,13 @@
 
 import uuid
 
-from typing import (  # pylint: disable=unused-import
-    Union, Optional, Any, TypeVar, TYPE_CHECKING
-)
+from typing import Any, Optional, TypeVar, Union, TYPE_CHECKING
```

Comment:
Started reorganizing imports than realized this was client code.. So will do it later 😄